R/auto.R

```diff
@@ -291,6 +293,17 @@ check_post_release <- function() {
 
 gh_scopes <- function() {
   out <- attr(gh::gh("/user"), "response")$"x-oauth-scopes"
-  if (out == "") return(character())
+  if (out == "") {
+    return(character())
+  }
   strsplit(out, ", *")[[1]]
 }
+
+check_gitignore <- function(files) {
+  files_ignored <- map_chr(files, ~ system2("git", c("check-ignore", .x), stdout = TRUE))
```

Comment:
Running `check-ignore -q` without `stdout = TRUE` seems slightly safer.

Reply:
The return value of `check-ignore` is a bit ugly. 

Both

`system2("git", c("check-ignore", "cran-comments.md"), stdout = FALSE)` and 
`system2("git", c("check-ignore","-q", "cran-comments.md"), stdout = FALSE)` 

return nothing if `cran-comments.md` is in `.gitignore`.

However, what we want is to detect this case.

Hence, we call `system2("git", c("check-ignore", "cran-comments.md"), stdout = TRUE)` which returns `"cran-comments.md"`. 

I do not see how `system2("git", c("check-ignore","-q", "cran-comments.md"), stdout = FALSE)` should work here?

Reply:
Without `stdout = FALSE`, an exit code is returned (invisibly).

```suggestion
  files_ignored <- map_int(files, ~ system2("git", c("check-ignore", "-q", .x), stdout = FALSE))
```